virtualfish/environment.fish

```diff
@@ -1,6 +1,30 @@
+# Set up the configurable filename
+if not set -q VIRTUALFISH_ENVIRONMENT_FILE
+    set -g VIRTUALFISH_ENVIRONMENT_FILE .env
+end
+
+function __vfsupport_set_env_file_path --description "Set VIRTUALFISH_ENVIRONMENT_FILE_PATH to appropriate value"
+    set -l vf_base (basename $VIRTUAL_ENV)
+    set -l vf_path $VIRTUAL_ENV/$VIRTUALFISH_ENVIRONMENT_FILE
+    # Check if projects plugin is used
+    if set -q PROJECT_HOME
+        set -l project_path $PROJECT_HOME/$vf_base/$VIRTUALFISH_ENVIRONMENT_FILE
+        if not test -r $project_path
+            and test -r $vf_path
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+        else
+            set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $project_path
+        end
+    else
+        set -g VIRTUALFISH_ENVIRONMENT_FILE_PATH $vf_path
+    end
+end
```

Comment:
I like your echo suggestion a lot - that does exactly what I was searching for. The reason I used more complicated logic when the Projects plugin is present was for creating a new env file. Here's how my thinking goes:
1. If we aren't using Projects, place the `.env` file in the virtualenv directory
2. If we are using Projects and there is a `.env` in the project directory, use that file
3. If we are using Projects and there is _not_ a `.env` file in the project directory and there _is_ and `.env` file in the virtualenv directory use that.
4. (This is why it got a little more complicated) If we are using Projects and there isn't a `.env` file in either the project or virtualenv directories, create a new file in the project directory.
Case 4 is why I have the logic like this:
```
if set -q PROJECT_HOME
    if not test -r $proj_env
        and test -r $vf_env
        # Use the virtualenv only when that's been established with a file there
    else
        # Prefer project directory
    end
else
    # Not using Projects
end
```
I think adding some comments and perhaps reordering the logic might make it a bit more clear.